src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exceptions.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.models.Manufacturer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public List<Manufacturer> getAll() {
+        String sqlRequest = "SELECT * FROM manufacturers WHERE is_deleted = FALSE;";
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.createConnection();
+                     PreparedStatement statement = connection.prepareStatement(sqlRequest)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(getManufacturer(resultSet));
+            }
+            return manufacturers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't retrieve records from DB", e);
+        }
+    }
+
+    public Optional<Manufacturer> get(Long id) {
+        String sqlRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = FALSE;";
```

Comment:
```suggestion
        String query = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = FALSE;";
```
when we talk about DB we call it query rather than request